tests/e2e/profile/profile.spec.js

```diff
@@ -0,0 +1,22 @@
+import { test } from "../../../src/fixtures/userGaragePage.fixture.js";
+import { USER_RESPONSE_BODY } from "../profile/fixtures/profileResponse.js";
+
+test("profile page should display changed user name", async ({
+  userProfilePage,
+}) => {
+  const { page } = userProfilePage;
+  await page.route("/api/users/profile", (route) => {
+    route.fulfill({ body: JSON.stringify(USER_RESPONSE_BODY) });
+  });
+
+  await userProfilePage.navigate();
+  await page.waitForSelector("div.panel-page_content > div > p");
+
+  const userDataText = await page.$eval(
```

Comment:
please forget about $eval if you can do it with built-in assertions
https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text